### PR TITLE
[Merged by Bors] - feat(FieldTheory/Adjoin): add `exists_finset_of_mem_adjoin`

### PR DESCRIPTION
**Pull request by acmepjz**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by erdOne** [comment]
maintainer merge

File: Mathlib/FieldTheory/Adjoin.lean
Comment (alreadydone) [inline Mathlib/FieldTheory/Adjoin.lean:809]: slight golf if you'd like:
```suggestion
  refine ⟨s.image Subtype.val, by simp, SetLike.le_def.mp ?_ hx'⟩
  simp_rw [Finset.coe_image, iSup_le_iff, adjoin_le_iff]
  rintro _ h _ rfl
  exact subset_adjoin F _ ⟨_, h, rfl⟩
```

**Review by alreadydone — APPROVED** [review]

**Comment by jcommelin** [comment]
Please merge @alreadydone's suggestion. After that, and after CI passes, please merge the PR.

bors d+

**Pull request closed without merging**
